src/API/Router.ts

```diff
@@ -74,7 +78,7 @@ export class Router {
     this.koa.use(KoaCors({ expose: ['X-Total-Count'] }))
     this.koa.use(LoggerMiddleware(this.logger))
     this.koa.use(HttpExceptionsMiddleware)
-    this.koa.use(KoaBody({ textLimit: 1000000 }))
+    this.koa.use(KoaBody({ textLimit: 1000000, multipart: true }))
```

Comment:
Don't think this is needed now, but we should consider some form of streaming upload here for music and other large format forms. Multipart isn't the best option here I think.

Reply:
@krobi64 do you know of anything out there that handles streams without multipart? or information on other ways? I couldn't find anything.

Reply:
Do we use an http server other than the library in node?

Reply:
not at the moment.